### DISCUSSION TITLE
Upgrade Issues

### DISCUSSION THREAD

**Discussion by goochusmc** [mkdocs/mkdocs#3338] [Q&A]
I tried upgrading from version 1.4.3 to the latest version today (1.5.2) and I encountered a few issues. I am using the mkdocs theme with the menu along the top and the menu and search options no longer work. I can get to pages if I put the url in manually but no drop downs or anything from the top level menu items and no response from the search (no search box even comes up). I also had an issue with my tables not rendering as they had previously, they look all screwed up in the latest version as compared to the previous. The tables used to have nicely spaced columns and rows alternating light and dark and now they just look like a jumbled mess.

I wanted to be able to use the new "not in nav" feature for an example of why I wanted to upgrade.

I stopped digging around to see if there were other problems at this point and rolled back my mkdocs version to 1.4.3 but am wondering if these are known issues related to changes in this release. I did not see anything in the release notes that would indicate these things have changed.

**Comment by oprypin** [comment]
> I am using the mkdocs theme with the menu along the top and the menu and search options no longer work. I can get to pages if I put the url in manually but no drop downs or anything from the top level menu items and no response from the search (no search box even comes up).

https://www.mkdocs.org/ uses the "mkdocs" theme too and the latest release of MkDocs. As you can see, it doesn't have any such problems. So in order to know anything about your particular problem, we'll need to see the particular site that is affected.

One thing to try is to refresh in your browser using Ctrl+F5, to clear the cache. Though if that *does* solve the issue, this is still concerning.

> I also had an issue with my tables not rendering as they had previously, they look all screwed up in the latest version as compared to the previous. The tables used to have nicely spaced columns and rows alternating light and dark and now they just look like a jumbled mess.

If you really want to get to the bottom of the issue, you should provide at the very least an example of your inputs. Otherwise this is an empty complaint that just brings sadness and nothing else.
I could guess that this is in fact about upgrading the version of "Markdown" rather than the version of "mkdocs" but in the end it has to be merely a guess.

> am wondering if these are known issues related to changes in this release

No, in fact I am very quick to fix any known regression, so there's no way for them to remain known and not fixed.

> I did not see anything in the release notes that would indicate these things have changed.

The release notes do contain a warning about the version of "Markdown", which is likely partly involved here.

Anyway, to get to the bottom of this you can try to upgrade *only* the version of "mkdocs" and nothing else, and then see if that's actually what caused it, or perhaps an upgrade of some other package. Then for particular issues please provide `pip freeze` outputs before and after, the inputs that are problematic and in which way they are problematic (expected vs actual).

**Comment by goochusmc** [comment]
Thanks for the quick response. Unfortunately I can't give you access to the site due to the contract under which work is being done but I can tell you we did not update anything other than mkdos from 1.4.3 to 1.5.2 and mkdocs-multirepo-plugin from 0.6.1 to 0.6.2. The issues I am seeing don't seem to be related at all to the imported pages that are generated using mkdocs-multirepo-plugin but I will try this without updating that as well.  This was all done on a VM in isolation before we tried doing anything on our live setup. this is how we have typically developed and we have changed versions previously without issues, we started originally with mkdocs 1.3.1. I will also try the cache clearing you mentioned to see if that helps and if not I will see if I can repeat the issue with something I can share with you.
